conf/application.conf

```diff
@@ -159,4 +159,9 @@ upscan {
   acceptedMimeTypes = "image/jpeg,image/png,image/tiff,application/pdf,text/plain,application/vnd.ms-outlook,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/vnd.oasis.opendocument.text,application/vnd.oasis.opendocument.spreadsheet,application/vnd.ms-powerpoint,application/vnd.openxmlformats-officedocument.presentationml.presentation,application/vnd.oasis.opendocument.presentation"
   checkInterval = 500millis
   timeout = 2seconds
+}
+
+timemachine{
+    enabled = true
+    date = "now"
```

Comment:
can you also update environment app-configs so

1. app-config-base to have timemachine.enabled defaulted to false
2. app-config-qa and app-config-staging timemachine enabled with date "now"
3. app-config-production to have defaulted false (a second guard).

This needs to be done for both services